src/CommunityStore/Utilities/Installer.php

```diff
@@ -115,7 +115,7 @@ public static function setConfigValue($key, $value)
     }
     public static function installPaymentMethods(Package $pkg)
     {
-        self::installPaymentMethod('invoice', 'Invoice', $pkg, null, true);
+        self::installPaymentMethod('invoice', 'Rechnung', $pkg, null, true);
```

Comment:
No, although this is just the initial name of the payment method (it can be changed via the dashboard), the initial setting is intended to be in English.